eng/pipelines/mgmt-ci.yml

```diff
@@ -3,7 +3,10 @@ trigger:
     include:
       - master
   paths:
+    include:
+      - eng/pipelines/mgmt-ci.yml
     exclude:
+      - eng/
```

Comment:
I'm afraid this might not work.  I think the paths are processed by first including everything under `include`, then excluding everything under `exclude` (if `include` is not specified the default is `*`).

https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#pr-triggers

So I think this change would **only** trigger the pipeline on `eng/pipelines/mgmt-ci.yml` and no other files in the repo, which is definitely not what we want.

I think the only way to achieve what we want is to exclude everything under `eng/` **except** for `eng/pipelines/mgmt-ci.yml` (or something close enough to this).

Reply:
Ah, I think you're right. Perhaps it's enough to just exclude `eng/common` for the moment, as that was the source of toil I was trying to eliminate (overriding invalid tests on auto-created PRs). And then when we switch to include that will solve the problem for the rest of the files.